package-managers/nuget.md

```diff
@@ -89,24 +89,43 @@ To work with public feeds, we just leave the credentials empty and Rider will co
 
 ### NuGet CLI
 
-#### Public feed
+#### Private feed
 
 Details on how to find the feed URL can be found in the [Find the feed URL](#find-the-feed-url) section.
 
-#### Private feed
+#### Public feed
 
 Details on how to find the feed URL can be found in the [Find the feed URL](#find-the-feed-url) section.
 
 ### dotnet CLI
 
-#### Public feed
+In this section we are going to add our packages via the dotnet CLI. We assume that the dotnet CLI tools are installed. If this is not the case, our friends at Microsoft can help us with that via [this link](https://www.microsoft.com/net/core).
+
+#### Private feed
 
 Details on how to find the feed URL can be found in the [Find the feed URL](#find-the-feed-url) section.
 
-#### Private feed
+#### Public feed
 
 Details on how to find the feed URL can be found in the [Find the feed URL](#find-the-feed-url) section.
 
+When adding a package to our project via the `dotnet CLI` we can get a generated command from the package details in our feed.
+
+![dotnet add command](assets/dotnet-add-command.png)
+
+We'll copy this command for later use.
+
+With our command on our clipboard, we open our favorite command line tool, in this case the standard Windows Command Prompt and navigate to the root of our project where we want to add the package to (where the `.csproj` file lives). There we paste the command from our clipboard and execute it.
+
+![dotnet add command in prompt](assets/dotnet-add-command-prompt.png)
+After running this command we get output like this
+
+![dotnet add command output](assets/dotnet-add-command-prompt-output.png)
+Our package is installed. To verify this we can look in the Solution File

```

Comment:
In the project file, we can verify our package is installed. The `dotnet add` command will have added a `PackageReference` element.